common/src/main/java/com/wynntils/features/user/overlays/CustomBarsOverlayFeature.java

```diff
@@ -350,31 +367,13 @@ public String icon() {
             return "۞";
         }
 
-        @Override
-        protected void renderBar(PoseStack poseStack, float renderY, float renderHeight, float progress) {
-            Texture universalBarTexture = Texture.UNIVERSAL_BAR;
-            RenderUtils.drawColoredProgressBar(
-                    poseStack,
-                    universalBarTexture,
-                    textColor,
-                    this.getRenderX(),
-                    renderY,
-                    this.getRenderX() + this.getWidth(),
-                    renderY + renderHeight,
-                    0,
-                    0,
-                    universalBarTexture.width(),
-                    universalBarTexture.height(),
-                    progress);
-        }
-
         @Override
         public WynnBossBarUtils.BarProgress noProgress() {
             return WynnBossBarUtils.NO_AWAKENED_PROGRESS;
         }
     }
 
-    public static class FocusBarOverlay extends BaseBarOverlay {
+    public class FocusBarOverlay extends BaseBarOverlay {
```

Comment:
@Incompleteusern Why did you turn all nested classes into inner classes? Qodana points out that it was not needed.

Reply:
Oops, brain slipped.